modules/ROOT/pages/_partials/keycloak/7_downloading-configuration-file.adoc

```diff
@@ -0,0 +1,56 @@
+// For more information, see: https://redhat-documentation.github.io/modular-docs/
+
+[id='downloading-the-configuration-file-{context}']
+= Downloading the Configuration File
+
+The `mobile-services.json` file provides the information for your mobile app to communicate with services.
+After you change any configuration in the {mobile-developer-console}, it is important to update that file in your IDE.
+
+.Prerequisites
+
+* The configuration of your {mobile-client} in {mobile-developer-console} is up-to-date.
+* You have set up your mobile app development environment.
+
+.Procedure
+
+. Open your {mobile-client} in Mobile Developer Console.
+. Copy the `mobile-services.json` configuration to your clipboard.
+. Save the contents of the clipboard to a new file called `mobile-services.json`.
++
+NOTE: The `mobile-services.json` file is the link between your provisioned services on OpenShift and the mobile app you are developing. This file provides all required configuration to initialise the various SDKs and get them hooked up/connected to the back-end services.
+. Follow the platform-specific instructions:
+
+[role="primary"]
+.Android
+****
+Move mobile-services.json to the following location in your application project:
+
+`app/src/main/assets/mobile-services.json`
+****
+
+[role="secondary"]
+.iOS
+****
+Move mobile-services.json to the following location in your application project:
+
+`<app directory>/mobile-services.json`
+
+NOTE: Ensure that `mobile-services.json` is a member of the project in the Xcode Project Navigator.
+****
+
+[role="secondary"]
+.Cordova
+****
+Move mobile-services.json to the following location in your application project:
+
+`src/mobile-services.json`
```

Comment:
@finp there is no folder named 'src' in a Cordova project:

---
[jstaffor@jstaffor test_cordova_project]$ cordova -v
8.1.2 (cordova-lib@8.1.1)

[jstaffor@jstaffor ~]$ cordova create test_cordova_project

[jstaffor@jstaffor ~]$ cd test_cordova_project/

[jstaffor@jstaffor test_cordova_project]$ ls
**config.xml  hooks  package.json  platforms  plugins  res  www**

I also took a look at the folder for hidden files and no sign of the 'src' folder.

Reply:
related to #551

Reply:
@pmacko1 yes, it is related to the #551 - `src/` is for ionic apps, `www/` for cordova

Reply:
@jstaffor  was this resolved?

Reply:
@finp  - it wasn't. I've created a Jira to follow up on this: https://issues.jboss.org/browse/AGMS-59